crates/forge/src/cmd/create.rs

```diff
@@ -120,7 +165,64 @@ impl CreateArgs {
             project.find_contract_path(&self.contract.name)?
         };
 
-        let output = compile::compile_target(&target_path, &project, shell::is_json())?;
+        let output: foundry_compilers::ProjectCompileOutput =
+            compile::compile_target(&target_path, &project, shell::is_json())?;
+
+        match get_child_contracts(output.clone(), &self.contract.name) {
```

Comment:
this introduces new codepath even for solidity contracts, ie should break compat between `resolc` and non `resolc` versions.